src/modules/storage/LedgerStorage.ts

```diff
@@ -1509,6 +1509,43 @@ export class LedgerStorage extends Storages
         });
     }
 
+    /**
+     * Returns UTXO's information about the UTXO hash array.
+     * @param utxos The array of UTXO hash
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the array of UTXO
+     * and if an error occurs the `.catch` is called with an error.
+     */
+    public getUTXOs (utxos: Array<Hash>): Promise<any[]>
+    {
+        let u = utxos.map(m => `x'${m.toBinary(Endian.Little).toString("hex")}'`);
+        let sql_utxo =
+            `SELECT
+                O.utxo_key as utxo,
+                O.amount,
+                O.lock_type,
+                O.lock_bytes,
+                T.block_height,
+                B.time_stamp as block_time,
+                O.type
+            FROM
+                tx_outputs O
+                INNER JOIN transactions T ON (T.tx_hash = O.tx_hash)
+                INNER JOIN blocks B ON (B.height = T.block_height)
```

Comment:
Why import from `outputs` without referencing the `utxos` table?

Reply:
Because there may be cases where the data we are looking for is not in Table utxos.
On second thought, however, I think we need to let them know when we don't have it.
Therefore, I am planning to change the form of the return value to table utxos.